lib/Runtime/Base/jitprofiling.cpp

```diff
@@ -6,6 +6,12 @@
 
 #ifdef VTUNE_PROFILING
 
+// For Linux getenv is been defined as PAL_getenv in pal/inc/pal.h
+// We need undef here so that VTune can use it to get environment variable 
+#ifdef PLATFORM_UNIX
+#undef getenv    
```

Comment:
I like this idea since it fixed the bug in misc.cpp. I will make the changes, run tests and update the PR.